### ISSUE TITLE
Allow metadata enum variants to choose a color & icon

### ISSUE THREAD

**Issue by gwennlbh** [cigaleapp/cigale#374]
icon should be required when choosing a color (for a11y reasons)

these would be also shown when displaying cascades (see #373)
